client/src/components/ElementEditor/ElementActions.js

```diff
@@ -17,6 +17,25 @@ class ElementActions extends Component {
     this.handleEditTabsClick = this.handleEditTabsClick.bind(this);
   }
 
+  /**
+   * Flattens a nested array
+   * Polyfill for Array.flat() which isn't available for jest tests (or IE11)
+   *
+   * @param {Array} arr
+   * @return {Array}
+   */
+  flatten(arr) {
```

Comment:
Could/should we check if `Array.flat()` is available first and use that and only use the custom implementation when it isn't?

Reply:
Just IE11 is missing Array.flat() - https://caniuse.com/array-flat

Reply:
We would need to add silverstripe-cms @ 4.9 as the minimal required version in that case?